torch/utils/tensorboard/writer.py

```diff
@@ -837,6 +837,26 @@ def add_custom_scalars(self, layout):
         """
         self._get_file_writer().add_summary(custom_scalars(layout))
 
+    def add_mesh(self, tag, vertices, colors=None, faces=None, config_dict=None, global_step=None, walltime=None):
+        """Add mesh or point cloud data to summary.
+
+        Args:
+            tag (string): Data identifier
+            vertices(torch.Tensor): List of the 3D coordinates of vertices.
+            colors(torch.Tensor): Colors for each vertex
+            faces(torch.Tensor): Indices of vertices within each triangle.
+            global_step (int): Global step value to record
+            walltime (float): Optional override default walltime (time.time())
+              seconds after epoch of event
+            config_dict: Dictionary with ThreeJS classes names and configuration.
+
+        Shape:
+            vertices: :math:`(N, T, 3)`.
+            colors: :math:`(N, T, 3)`. The values should lie in [0, 255] for type `uint8` or [0, 1] for type `float`.
+            vertices: :math:`(N, T, 3)`. The values should lie in [0, numVerts] for type `uint8`.
+        """
```

Comment:
Please add an `Examples::` section like

```
        Examples::
            from torch.utils.tensorboard import SummaryWriter
            # setup data....
            writer = SummaryWriter()
            writer.add_mesh('my_mesh', ...)

            writer.close()
```

@lanpa do you want to do this?

Reply:
I'm going to do it.

Is porting this sufficient? https://github.com/tensorflow/graphics/blob/master/tensorflow_graphics/g3doc/tensorboard.md#summary-api

Reply:
So long as the code is runnable that would be great.

Reply:
Added example code, this is how it looks:

![Screenshot from 2019-05-21 22-24-53](https://user-images.githubusercontent.com/1162980/58128318-a2447c00-7c17-11e9-8131-eb0feeaceef0.png)